packages/components/package.json

```diff
@@ -48,6 +48,7 @@
         "@typescript-eslint/eslint-plugin": "^5.45.1",
         "@typescript-eslint/parser": "^5.45.1",
         "autoprefixer": "^10.4.13",
+        "braces": "^3.0.3",
```

Comment:
I'm unsure if it's good practice, or not to add the dependencies directly here, instead of updating the packages that depend on them.

Reply:
Hmm, how could we do that?